### PR TITLE
Update docs for expectException

### PR DESCRIPTION
**Pull request by Rowlando13**
Add clarity for how expectException method should be called.

### CODE REVIEW

**Comment by Rowlando13** [comment]
First PR for this project. Let me know what I need to fix.

File: src/writing-tests-for-phpunit.rst [hunk 1]
```diff
@@ -646,7 +646,7 @@ whether an exception is thrown by the code under test.
     {
         public function testException(): void
         {
-            $this->expectException(InvalidArgumentException::class);
+            $this->expectException(InvalidArgumentException::class);
```
Comment (peter279k) [inline src/writing-tests-for-phpunit.rst:649]: I think the additional space should be removed.

Comment (Rowlando13) [reply src/writing-tests-for-phpunit.rst]: Fixed.

**Comment by sebastianbergmann** [comment]
Thank you for your contribution. I appreciate the time you invested in preparing this pull request. However, I have decided not to merge it.

**Comment by Rowlando13** [comment]
@sebastianbergmann You are very welcome. Is there any particular reason it did not get merged? I would like to know so that I can make better contributions in the future. My only though was that I was documenting something that was a bit obvious.

**Comment by sebastianbergmann** [comment]
I rewrote large parts of the documentation while updating it for PHPUnit 10 and this PR was no longer applicable after that.

**Comment by Rowlando13** [comment]
@sebastianbergmann Thank you for clarifying!

**Pull request closed without merging**